src/modules/quick_input_parser_module.f90

```diff
@@ -14,70 +14,67 @@ module quick_input_parser_module
 
     contains
 
-        subroutine read_float_keyword(keywdline, keyword, val)
+        subroutine checkformat(i,line,keyword)
             implicit none
-            character(len=*), intent(in) :: keywdline
+            integer, intent(inout) :: i
+            character(len=*), intent(in) :: line
+            character(len=*), intent(in) :: keyword
+
+            i = index(line, trim(keyword)//trim('='))
+            if(i==0) then
+                call PrtErr(OUTFILEHANDLE, "USE keyword=val format in input")
```

Comment:
I am not sure what PrtErr() does. But perhaps we should indicate which keyword has an issue by adding `call PrtErr(OUTFILEHANDLE, "Error with keyword "//trim(keyword)//" encountered.")`

Reply:
Just prints the error: https://github.com/Altaimorin/QUICK/blob/bugfix113/src/subs/PrtErr.f90. Actually this is not consistent with our new error handling implementation. I will have to fix this once this PR and PR #85 is merged.